org.kde.okular.json

```diff
@@ -0,0 +1,223 @@
+                    "url": "https://download.kde.org/stable/applications/18.04.0/src/libkexiv2-18.04.0.tar.xz", 
+                    "sha256": "fec30c8a4d9e0cfd3a00f980c7add91f885f997cc7ff3fe4b8dbef268d5acd87" 
+                } 
+            ]
+        },
+        {
+            "name": "openjpeg",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "cleanup": [
+                "/bin",
+                "/lib/openjpeg-*",
+                "*.a"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/uclouvain/openjpeg/archive/v2.3.0.tar.gz",
+                    "sha256": "3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a"
+                }
+            ]
+        },
+        {
+            "name": "popplerdata",
+            "no-autogen": true,
+            "make-install-args": [
+                "prefix=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://poppler.freedesktop.org/poppler-data-0.4.8.tar.gz",
+                    "sha256": "1096a18161f263cccdc6d8a2eb5548c41ff8fcf9a3609243f1b6296abdf72872"
+                }
+            ]
+        },
+        {
+            "name": "poppler",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+                "-DCMAKE_INSTALL_INCLUDEDIR=/app/include"
+            ],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://poppler.freedesktop.org/poppler-0.64.0.tar.xz",
+                    "sha256": "b21df92ca99f78067785cf2dc8e06deb04726b62389c0ee1f5d8b103c77f64b1"
+                }
+            ]
+        },
+        {
+            "name": "qmobipocket",
+            "config-opts": ["-DCMAKE_BUILD_TYPE=Release", "-DCMAKE_INSTALL_LIBDIR=lib"],
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "archive", 
+                    "url": "https://download.kde.org/stable/applications/18.04.0/src/kdegraphics-mobipocket-18.04.0.tar.xz", 
+                    "sha256": "16baaf08af6a2415cbcf1e68c7547d1bf0d4be0a6d79a8a9862a6becc704e005" 
+                } 
+            ]
+        },
+        {
+            "name": "libzip",
+            "buildsystem": "cmake-ninja",
+            "sources": [ 
+                { 
+                    "type": "archive", 
+                    "url": "https://libzip.org/download/libzip-1.4.0.tar.xz", 
+                    "sha256": "e508aba025f5f94b267d5120fc33761bcd98440ebe49dbfe2ed3df3afeacc7b1" 
+                } 
+            ]
+        },
+        {
+            "name": "chmlib",
```

Comment:
this needs a reconfigure for aarch64 https://flathub.org/builds/#/builders/41/builds/95